helm_deploy/cla-backend/templates/cron.yaml

```diff
@@ -77,4 +77,24 @@ spec:
             env:
               {{ include "cla-backend.app.vars" . | nindent 12 }}
           restartPolicy: OnFailure
----
\ No newline at end of file
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-fix-missing-outcome-codes
+  labels:
+    {{- include "cla-backend.labels" . | nindent 4 }}
+spec:
+  schedule: "0 5 4 * *"
```

Comment:
Just a query on this, please dismiss this I am mistaken. 

However, this CronJob is going to run on the 4th of each month at 5AM, however, the ticket says "The task has been scheduled to run before the 4th of each month.".

Is it okay if the task runs before the start of the work day, or could this be an issue? Is there a reason why this shouldn't be run on the 3rd of each month at 23:00, for example?

Reply:
A good question. 4th and after is when they run their big reports. By doing on the 4th in the morning, they'll have a fresh (and anything was missing would be filled) database with no missing codes. This leave little space for anyone to add more cases that could have blanks in them.